CHANGELOG.rst

```diff
@@ -1,6 +1,12 @@
 Changelog
 =========
 
+2.10.1 (2020-06-??)
```

Comment:
imho this should be a pytest-cov 3 release

Reply:
Because it removes some `compat.xxx` names?

If compatibility with pytest-xdist <= 1.22 (from 2018) is really that important - and note that the oldest version pytest-cov is tested against is xdist 1.27 (from 2019) - I could add them back.  @ionelmc?

Reply:
there's no harm in bumping the major version, but we don't have to make the decision in this PR

Reply:
We certainly don't want to extend the testing grid, it's way too big already.

Reply:
As expected, old releases like pytest-xdist 1.22 account for very few downloads:

![image](https://user-images.githubusercontent.com/1324225/84644964-986b7600-af08-11ea-9384-f3d6d9a90a9e.png)

[pepy.tech link](https://pepy.tech/project/pytest-xdist?versions=*&versions=1.32.*&versions=1.31.*&versions=1.30.*&versions=1.29.*&versions=1.28.*&versions=1.27.*&versions=1.26.*&versions=1.25.*&versions=1.24.*&versions=1.23.*&versions=1.22.*)